### ISSUE TITLE
How to calculate area of a polygon?

### ISSUE THREAD

**Issue by anshukishore** [GEOSwift/GEOSwift#64]
I have created a GEOSwift polygon from a geojson object. I want to calculate it's area. I do not see a method exposed. Is there a GeoSwift function to do that?

**Comment by vfn** [comment]
Hi @anshukishore, unfortunately area has not been implemented yet.

It would be awesome if you could add it, as well as some tests, and open a PR.

You can have a look at [MiscFunctions.swift](https://github.com/GEOSwift/GEOSwift/blob/master/GEOSwift/MiscFunctions.swift), the first function on that todo list is the area. The implementation should be similar to the distance function implemented on that file. Although I think `area` can be a var as it does not require any parameter.

Something like:
```Swift
    var area: Double {
        var area: Double = 0
...
        return area
    }
```

**Comment by anshukishore** [comment]
Thank you for the response. I will consider an implementation.

**Comment by simonbilskyrollins** [comment]
Looks like area was implemented in #94

**Comment by vfn** [comment]
@simonbilskyrollins that's correct! Thank you for pointing that out.
